library/src/main/kotlin/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -68,25 +71,28 @@ public class ChuckerInterceptor private constructor(
     override fun intercept(chain: Interceptor.Chain): Response {
         val transaction = HttpTransaction()
         val request = chain.request()
-
-        requestProcessor.process(request, transaction)
-
+        val shouldProcessTheRequest = !skipPaths.any { it == request.url.encodedPath }
+        if(shouldProcessTheRequest){
+            requestProcessor.process(request,transaction)
+        }
         val response = try {
             chain.proceed(request)
         } catch (e: IOException) {
             transaction.error = e.toString()
             collector.onResponseReceived(transaction)
             throw e
         }
-
-        return responseProcessor.process(response, transaction)
+        return if(shouldProcessTheRequest)
+            responseProcessor.process(response,transaction)
+        else response
```

Comment:
Let's merge it as it is, but I believe KtLint is broken here as this is not properly formatted